apps/ios/Shared/Views/UserSettings/ProtocolServersView.swift

```diff
@@ -119,11 +124,22 @@ struct SMPServersView: View {
                 )
             }
         }
+        .onAppear {
```

Comment:
didn't we have issues with loading data in onAppear? I think they happened sporadically / were hard to reproduce.. and maybe there was a better fix than to replace onAppear with a parameter so it's ok..